### PR TITLE
refactoring/tests `ng add`

### PR DESCRIPTION
**Pull request by d-koppenhagen**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [x] Other... Please describe: Additional tests

## What is the current behavior?

`ng-add` didn't work (0.0.10)

## What is the new behavior?

Verification that other schematics will be called when running `ng-add`

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by d-koppenhagen** [comment]
@jorgeucano please have a look at it. It should harden the `ng-add` behaviour. Tests are running again locally. (Worked on my machine- all tests were green) 😄

File: schematics/scully/src/scully/index_spec.ts [hunk 1]
```diff
@@ -39,5 +39,14 @@ describe('scully schematic', () => {
       expect(scripts.scully).toEqual('scully');
       expect(scripts['scully:serve']).toEqual('scully serve');
     });
+
+    it(`should not override an existing cully config file'`, async () => {
```
Comment (aaronfrost) [inline schematics/scully/src/scully/index_spec.ts]: scully, not cully

**Comment by d-koppenhagen** [comment]
@aaronfrost should be ready to merge now

File: schematics/scully/src/ng-add/index.ts [hunk 2]
```diff
@@ -134,17 +135,21 @@ const injectIdleService = (options: Schema) => (tree: Tree, context: SchematicCo
   }
 };
 
-const runScullySchematic = (options: Schema) => (tree: Tree, context: SchematicContext) => {
+const runBlogSchematic = (options: Schema) => (tree: Tree, context: SchematicContext) => {
```
Comment (jorgeucano) [inline schematics/scully/src/ng-add/index.ts]: We decide remove the blog option, as a promp in the `ng add` can you remove the option? please

Comment (d-koppenhagen) [reply schematics/scully/src/ng-add/index.ts]: Can we may merge this one and #177 first? As it will cause some conflicts otherwise I think

Comment (d-koppenhagen) [reply schematics/scully/src/ng-add/index.ts]: In #177 I added some tests as well. So I would prefer to remove the option in #177 or to remove it in an extra PR / issue.

**Pull request merged**